pom.xml

```diff
@@ -2301,6 +2301,8 @@ under the License.
 								<exclude>org.apache.flink.streaming.api.environment.StreamExecutionEnvironment#fromData(java.lang.Class,java.lang.Object[])</exclude>
 								<exclude>org.apache.flink.streaming.api.environment.StreamExecutionEnvironment#fromData(java.util.Collection)</exclude>
 								<exclude>org.apache.flink.streaming.api.environment.StreamExecutionEnvironment#fromData(java.util.Collection,org.apache.flink.api.common.typeinfo.TypeInformation)</exclude>
+								<exclude>org.apache.flink.api.common.functions.RuntimeContext</exclude>
+								<exclude>org.apache.flink.api.connector.source.SourceReaderContext</exclude>
```

Comment:
This should not be a hotfix commit. These change does not make sense by itself without the subsequent commits.

Reply:
I have merged these changes to the commit where it should be.